plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -354,6 +354,73 @@ var _ = Describe("validator", func() {
 			admissionHandler.SetAuthorizer(auth)
 		})
 
+		Context("cluster autoscaler configuration check for ScaleDownUnneededTime and ScaleDownDelayAfterAdd time", func() {
+			BeforeEach(func() {
+				shoot.Spec.Kubernetes.ClusterAutoscaler = &core.ClusterAutoscaler{}
+			})
+
+			DescribeTable("validate cluster autoscaler configuration on shoot create", func(workerUnneededTime, unneededTime, afterAddTime *metav1.Duration, errorMatcher types.GomegaMatcher) {
```

Comment:
I think it is better to pass worker2's unneeded Time as a parameter to `DescribeTable` function